tests/New_ATS/pylib/testlist.json

```diff
@@ -6,12 +6,16 @@
       "enum-oneof",
       "message-oneof",
       "multiple-field-oneof",
+      "all-datatypes-oneof",
       "nested-oneof",
       "sibling-oneof",
-      "simple-oneof"
+      "simple-oneof",
+      "oneof-streaming",
+      "enum-streaming"
     ],
     "gen_type": "0",
     "labview_version": "2019",
-    "labview_bitness": "32"
+    "labview_bitness": "32",
+    "clean_gen": true
```

Comment:
Update the read me to include this flag, mention that its always clean gen by default